Reference/Angular/Services/editorService/Index.md

```diff
@@ -0,0 +1,83 @@
+# Editor Service
+
+In Umbraco 8, the Angular `editorService` service is the primary resource used for opening overlays and handling infinite editing. Besides the `open` and `close` functions, the service also contains functions for opening specialized overlays/editors - eg. `contentPicker` or `mediaPicker`.
```

Comment:
I've suggested removing the mention of versions in the text. Instead, we could perhaps add some YAML tags to article, about from which Umbraco version the information here in valid.

What do you think?

```suggestion
The Angular `editorService` service is the primary resource used for opening overlays and handling infinite editing. Besides the `open` and `close` functions, the service also contains functions for opening specialized overlays/editors - eg. `contentPicker` or `mediaPicker`.
```